### DISCUSSION TITLE
where to get apiKey

### DISCUSSION THREAD

**Discussion by yewkay** [code-chat-br/whatsapp-api#142] [Q&A]
My codechat is hosted by docker with port  8084 I might have missed something obvious, when i do a 
curl -L -g -X POST 'http://127.0.0.1:8084/instance/create' \
-H 'Content-Type: application/json' \
-H 'Accept: application/json' \
--data-raw '{
  "instanceName": "codechat-bot",
  "description": "Instance: Test V1"
}'

i got {"status":403,"error":"Forbidden","message":["Missing global api key","The global api key must be set"]}

where do i get the api key i how do i pass it to codechat?

thank you

**Comment by Hamoo17** [comment]
You can find The API Global Key on .env File 
`AUTHENTICATION_GLOBAL_AUTH_TOKEN=*************************************************`
